### ISSUE TITLE
Improve discoverability of keyboard shortcuts for highlighting and annotating

### ISSUE THREAD

**Issue by lyzadanger** [hypothesis/client#4742]
> Keyboard shortcuts for highlighting and annotating are not conveyed to the user via instruction

**Comment by lyzadanger** [comment]
First area of discovery: would it be feasible to use an `aria-live` region within the current Adder?

**Comment by lyzadanger** [comment]
Putting a little more flesh on the bones of the idea of an `aria-live` region with adder-related help. One way this could be done is:

* Render an additional component along with the `AdderToolbar` in `adder`.
* This component could render a non-visible `aria-live` region, and manage state as to whether the adder has ever been shown (it would also take a visibility boolean prop like the `AdderToolbar`.
* The first time that visibility prop goes to `true`, content could be added to the `aria-live` region, which should be read out by screen readers.

There is a risk that this could be annoying to our power-screen-reader users, as this help text would be announced the first time the adder is shown on every page/document load, but I also imagine it's easy to silence the voiceover text and is likely something screen-reader users do all the time. But that's an assumption.
